openquake/hazardlib/gsim/mgmpe/cy14_site_term.py

```diff
@@ -0,0 +1,96 @@
+# The Hazard Library
+# Copyright (C) 2012-2020 GEM Foundation
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""
+Module :mod:`openquake.hazardlib.mgmp.nrcan15_site_term` implements
+:class:`~openquake.hazardlib.mgmpe.ask14_site_term.ASK14SiteTerm`
+"""
+
+import copy
+import numpy as np
+from openquake.hazardlib import const
+from openquake.hazardlib.gsim.base import GMPE, registry
+from openquake.hazardlib.gsim.chiou_youngs_2014 import ChiouYoungs2014
+
+
+class CY14SiteTerm(GMPE):
+    """
+    Implements a modified GMPE class that can be used to account for local
+    soil conditions in the estimation of ground motion.
```

Comment:
Could add to the end of this sentence: " using the site term from ChiouYoungs2014"